src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -149,21 +179,55 @@ public static class ClientOperations
                     using (var req = new HttpRequestMessage(HttpMethod.Get, "/headers") { Version = httpVersion })
                     using (HttpResponseMessage m = await ctx.SendAsync(req))
                     {
-                        ValidateResponse(m, httpVersion);
+                        ValidateHttpVersion(m, httpVersion);
+                        ValidateStatusCode(m);
                         ValidateContent(ctx.ContentSource, await m.Content.ReadAsStringAsync());
                     }
                 }),
 
+                ("GET Echo Headers",
```

Comment:
Do we need both this and the GET Headers above?  Could this replace the other one?

Reply:
Maybe, but they are slightly different, e.g. the existing one adds trailer headers. Probably good to keep some variation?

Reply:
Your test is adding a random number of headers, right?  Couldn't it also add a random number of trailers (including 0)?

Reply:
I can certainly add that.

Reply:
@stephentoub done.